src/utils/headerLinks.js

```diff
@@ -29,13 +29,20 @@ const HEADER_LINKS = [
     ],
   },
   {
-    href: 'work-orders/cautionary-alerts',
+    href: '/work-orders/cautionary-alerts',
     id: 'cautionary-alerts',
     description: 'Cautionary Alerts',
     permittedRoles: [OPERATIVE_ROLE],
   },
+
+  {
+    href: 'https://sites.google.com/hackney.gov.uk/repairshubsupport',
```

Comment:
Ideally this would sit in one of the `NEXT_PUBLIC_` environment variables. I don't think this prevents a merge though.